### PR TITLE
Track consignment submission flow steps

### PR DESCRIPTION
**Pull request by mikehrom**
Events for the steps described [here](https://docs.google.com/spreadsheets/d/1QtlyK1XGi1QWSqqqT4mMmZUOYMfPxHlOjupNYZ6LCGo/edit#gid=0) and [here](https://www.figma.com/file/mcQv2yvaXiacFKZbC74BE0/SWA-Submissions-Flow?node-id=2465%3A16433).

@abhitip  should we incorporate a context_module for these?

### CODE REVIEW

**Comment by abhitip** [comment]
Thanks @mikehrom! love kieran's email <3

I think we should add a context_module for `consignmentSubmissionFlow` for all the events - what do you think?

File: src/Schema/Events/Consignments.ts [hunk 1]
```diff
@@ -26,3 +26,113 @@ export interface ConsignmentSubmitted {
...
+ *  {
+ *    action: "viewArtworkMyCollection",
+ *    submission_id: "52521",
+ *    user_email: "kieranmbh@gmail.com"
+ *    user_id: "5cd6b173746be6109c86321d"
+ *  }
+ * ```
+ */
+export interface ViewArtworkMyCollection {
+  action: ActionType.viewArtworkMyCollection
```
Comment (abhitip) [inline src/Schema/Events/Consignments.ts:134]: let's also add a context_owner_type: `consignmentSubmissionThankYou` (in addition to the context_module)

File: src/Schema/Events/Consignments.ts [hunk 2]
```diff
@@ -26,3 +26,113 @@ export interface ConsignmentSubmitted {
...
+ *  {
+ *    action: "submitAnotherArtwork",
+ *    submission_id: "52521",
+ *    user_email: "kieranmbh@gmail.com"
+ *    user_id: "5cd6b173746be6109c86321d"
+ *  }
+ * ```
+ */
+export interface SubmitAnotherArtwork {
+  action: ActionType.submitAnotherArtwork
```
Comment (abhitip) [inline src/Schema/Events/Consignments.ts:112]: let's also add a context_owner_type: consignmentSubmissionThankYou (in addition to the context_module)

**Comment by mikehrom** [comment]
Yeah, I'll add those contexts and then we can merge.

**Comment by mikehrom** [comment]
@abhitip -- i added the above using existing context_module and owner_type options. look good?

**Comment by abhitip** [comment]
Looks good! Thanks @mikehrom

**Pull request merged**